x-pack/plugins/security_solution/public/detections/components/callouts/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export * from './callout_switcher';
+export * from './callout_types';
+export * from './callout';
```

Comment:
`x-pack/plugins/security_solution/public/detections/components/callouts`

- is this a right place for these components?
- worth moving to `common`?

Reply:
I'd say for something that has the reusability of a callout and isn't tied specifically to the Detections page would fit well in the `public/common` folder. That's where similar lightweight, reusable designs like our custom toasters reside

Reply:
👍  I moved the new reusable callouts to `public/common/components/callouts`.